docs/servo_power_test/src/power_test_controller.ts

```diff
@@ -48,43 +49,48 @@ export class PowerTestController {
   public async startMeasurement() {
     if (this.ui.configNum === 0) return;
     this.marginTime = Number(this.ui.marginTimeInput.value);
+    this.itrNum = parseInt(this.ui.itrInput.value);
+    if (this.itrNum <= 0) return;
     await this.servoController.servoShell.select();
     await this.runner.dut.select();
     await this.initializePort();
     await this.setConfig();
-    for (let i = 0; i < this.ui.configNum; i++) {
-      this.currentConfigNum = i;
-      console.log(`start running config${i}`);
-      await this.configList[i].start();
+    for (let currentItrNum = 0; currentItrNum < this.itrNum; currentItrNum++) {
```

Comment:
while the local variables should have a descriptive name, maybe single or two character name here is good for readability.
> Exception: Variables that are in scope for 10 lines or fewer, including arguments that are not part of an exported API, may use short (e.g. single letter) variable names.
https://google.github.io/styleguide/tsguide.html#descriptive-names